wrappers/node/package.json

```diff
@@ -64,6 +66,7 @@
     "jslint:fix": "standard --fix",
     "lint:demo": "standard demo/*",
     "doc-gen": "./node_modules/.bin/typedoc --out doc --excludePrivate --excludeProtected --ignoreCompilerErrors src",
+    "test:integration": "node demo/integration-test.js",
```

Comment:
Maybe we should include revocation as part of the tested scenario?

Reply:
Done